test/transforms.jl

```diff
@@ -424,24 +429,26 @@
       p = plot(p₁, p₂, p₃, p₄, layout=(2,2))
       q = plot(p₂, p₃, p₄, p₅, p₆, p₇, layout=(2,3))
 
-      @test_reference joinpath(datadir,"eigenanalysis-1.png") p
-      @test_reference joinpath(datadir,"eigenanalysis-2.png") q
+      @test_reference joinpath(datadir, "eigenanalysis-1.png") p
+      @test_reference joinpath(datadir, "eigenanalysis-2.png") q
+      # savefig(p, joinpath(datadir, "eigenanalysis-1.png"))
+      # savefig(q, joinpath(datadir, "eigenanalysis-2.png"))
```

Comment:
```suggestion
```

Reply:
I removed the comments of savefig function.